conda/meta.yaml

```diff
@@ -7,24 +7,27 @@ source:
     git_rev: v1.6.0
 
 build:
-    script: python setup.py install
+    number: 1
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
This is the preferred build command (see http://conda-forge.org/docs/maintainer/adding_pkgs.html#build).